diff --git a/node_modules/react-native-video-trimmer-ui/lib/commonjs/components/VideoTrimmer.js b/node_modules/react-native-video-trimmer-ui/lib/commonjs/components/VideoTrimmer.js
index 5648227..767d34c 100644
--- a/node_modules/react-native-video-trimmer-ui/lib/commonjs/components/VideoTrimmer.js
+++ b/node_modules/react-native-video-trimmer-ui/lib/commonjs/components/VideoTrimmer.js
@@ -81,7 +81,7 @@ function VideoTrimmerUI(props, ref) {
   };
   return /*#__PURE__*/(0, _jsxRuntime.jsxs)(_reactNative.View, {
     style: [_styles.videoTrimmerStyles.container, containerStyle],
-    children: [/*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeVideo.default, {
+    children: [/*#__PURE__*/(0, _jsxRuntime.jsx)(_reactNativeVideo.default, { playInBackground: true,
       style: _styles.videoTrimmerStyles.video,
       ref: videoRef,
       source: source,
diff --git a/node_modules/react-native-video-trimmer-ui/lib/module/components/VideoTrimmer.js b/node_modules/react-native-video-trimmer-ui/lib/module/components/VideoTrimmer.js
index ecc501d..e91e9aa 100644
--- a/node_modules/react-native-video-trimmer-ui/lib/module/components/VideoTrimmer.js
+++ b/node_modules/react-native-video-trimmer-ui/lib/module/components/VideoTrimmer.js
@@ -74,7 +74,7 @@ function VideoTrimmerUI(props, ref) {
   };
   return /*#__PURE__*/_jsxs(View, {
     style: [videoTrimmerStyles.container, containerStyle],
-    children: [/*#__PURE__*/_jsx(Video, {
+    children: [/*#__PURE__*/_jsx(Video, { playInBackground: true,
       style: videoTrimmerStyles.video,
       ref: videoRef,
       source: source,
